pkg/dao/repository_configs.go

```diff
@@ -548,6 +548,8 @@ func (r repositoryConfigDaoImpl) Fetch(ctx context.Context, orgID string, uuid s
 		}
 		contentURL := pulpContentURL(contentPath, repoConfig.LastSnapshot.RepositoryPath)
 		repo.LastSnapshot.URL = contentURL
+		repo.LatestSnapshotURL = pulpContentURL(contentPath,
+			fmt.Sprintf("%v/%v/%v", strings.Split(repo.LastSnapshot.RepositoryPath, "/")[0], repo.UUID, "latest"))
```

Comment:
ah yep good point! just pushed that up

Reply:
you can actually delete the 	ModelToApiFields(repoConfig, &repo)  above too

Reply:
if a repo doesn't have snapshotting enabled though, the response isn't formed properly. since that's already handled in convertToResponses, maybe i can call ModelToApiFields only if snapshotting isn't enabled?

Reply:
oh i could also just call convertToResponses whether snapshotting is enabled or not